AWSAuthSDK/Sources/AWSMobileClient/AWSMobileClientExtensions.swift

```diff
@@ -890,7 +890,9 @@ extension AWSMobileClient: UserPoolAuthHelperlCallbacks {
         } else {
             // If user is not signedIn, we reach here as part of the signIn flow. Next step
             // is to inform the user to enter custom auth challenge.
-            let result = SignInResult(signInState: .customChallenge, codeDetails: nil)
+            let result = SignInResult(signInState: .customChallenge,
+                                      parameters: customAuthentiationInput.challengeParameters,
```

Comment:
Fix spelling of `customAuthenticationInput`

Reply:
Fixed